jest.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: ['<rootDir>/jest-setup.ts'],
   moduleNameMapper: {
+    kbar: '<rootDir>/__mocks__/kbar.ts',
```

Comment:
So this just defines a "dummy" kbar instead of the real one?

Reply:
yep - I found that easier than working around the issues with it. haven't actually mocked it either - just overwritten the problematic hook